packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts

```diff
@@ -46,7 +46,7 @@ export class EntityEventsToDbListener {
 
   private async handle(batchEvent: WorkspaceEventBatch<ObjectRecordBaseEvent>) {
     const filteredEvents = batchEvent.events.filter(
-      (event) => event.objectMetadata?.isAuditLogged,
+      (event) => event.objectMetadata?.isAuditLogged && event.properties.after, // We ignore events on a destroyed record
```

Comment:
won't this filter out all destroy events?
I think we still want them

Reply:
I tried to follow this comment
https://github.com/twentyhq/twenty/issues/9358#issuecomment-2586764706
We should probably update the implement destroy event differently, wdyt @charlesBochet ?
Maybe a few more details in the ticket from you will help me follow up on this PR

Reply:
Hey @pacyL2K19 - I just pushed an alternative proposal, looking forward to your feedback @pacyL2K19 and @charlesBochet 

The previous solution didn't work because it affected all job types and also the delete event. We only want to disable the TIMELINE job for the DESTROYED event

Reply:
Nice one @FelixMalfait 
Just saw the changes, it look better